src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -1956,6 +1955,97 @@ void F()
                 "C: {<>c}");
         }
 
+        [Fact]
+        public void Lambda_Delete()
+        {
+            using var _ = new EditAndContinueTest()
+                .AddBaseline(
+                    source: """
+                        using System;
+
+                        class C
+                        {
+                            void F()
+                            {
+                                _ = new Action(() => Console.WriteLine(1));
+                                _ = new Action(<N:0>() => Console.WriteLine(2)</N:0>);
+                            } 
+                        }
+                        """,
+                    validator: _ =>
+                    {
+                    })
```

Comment:
Nit: I think its good to at least verify the table sizes, even if not their exact contents, so its clear that the changes in the next geneartion are modifying existing rows.